### ISSUE TITLE
tf.keras.ops.image.gaussian_blur appears to swap height and width sigma parameters

### ISSUE THREAD

**Issue by ALinrunrun** [tensorflow/tensorflow#127247]
### Issue type

Bug

### Have you reproduced the bug with TensorFlow Nightly?

Yes

### Source

source

### TensorFlow version

tf 2.21.0, also reproduced on tf 2.22.0-dev20260904

### Custom code

Yes

### OS platform and distribution

Linux Ubuntu 22.04

### Python version

Python 3.13.5

### Current behavior?

`tf.keras.ops.image.gaussian_blur` returns an incorrect forward value when distinct height and width sigma values are provided.

The API documents `sigma` in height,width order. In the reproducer below, `sigma=(0.5, 2.0)` is used with a nonsymmetric `3 x 3` image. Only the center pixel is inspected, so the result does not depend on padding behavior.

The expected center value using height sigma `0.5` and width sigma `2.0` is approximately `-1106.2184462981334`. However, TensorFlow returns `-1242.3213516003145`, which is consistent with applying the sigma values to the opposite axes.

Expected behavior?

`tf.keras.ops.image.gaussian_blur` should apply `sigma=(0.5, 2.0)` in height,width order. For the reproducer below, the center output should be close to `-1106.2184462981334`, not `-1242.3213516003145`.

### Standalone code to reproduce the issue

```shell
import os
os.environ["CUDA_VISIBLE_DEVICES"] = ""
os.environ["OMP_NUM_THREADS"] = "1"
os.environ["TF_NUM_INTRAOP_THREADS"] = "1"
os.environ["TF_NUM_INTEROP_THREADS"] = "1"

import tensorflow as tf

def target(t):
    image = tf.reshape(
        tf.stack([(i + 1) ** 2 * t - i for i in range(9)]),
        [3, 3, 1],
    )

    y = tf.keras.ops.image.gaussian_blur(
        image,
        kernel_size=(3, 3),
        sigma=(0.5, 2.0),
        data_format="channels_last",
    )

    return y[1, 1, 0]

x = tf.constant(-40.0, dtype=tf.float64)

actual = target(x)
expected = -1106.2184462981334

print("actual:", actual.numpy())
print("expected:", expected)

if abs(float(actual) - expected) > 1e-9:
    print("BUG REPRODUCED: tf.keras.ops.image.gaussian_blur appears to swap height and width sigma parameters")
else:
    print("not reproduced")
```

### Relevant log output

```shell
actual: -1242.3213516003145
expected: -1106.2184462981334
BUG REPRODUCED: tf.keras.ops.image.gaussian_blur appears to swap height and width sigma parameters
```

**Comment by jdymitarai** [comment]
Root cause identified and fix submitted in https://github.com/keras-team/keras/pull/23614.

In gaussian_blur, kernel_size and sigma are specified in (height, width) order. However, _get_gaussian_kernel2d across all backends (TensorFlow, PyTorch, JAX, NumPy, OpenVINO) assigned size[0], sigma[0] to kernel1d_x and size[1], sigma[1] to kernel1d_y, then combined them via outer(kernel1d_y, kernel1d_x) (or 	ensordot(kernel1d_y, kernel1d_x)).

Because dimension 0 of the resulting 2D convolution kernel corresponds to height (Y) and dimension 1 corresponds to width (X):
- Height received size[1], sigma[1] (the width parameters).
- Width received size[0], sigma[0] (the height parameters).

This inverted height and width sigma across all backends. PR https://github.com/keras-team/keras/pull/23614 fixes the mapping to outer(kernel1d_h, kernel1d_w) and adds unit tests confirming the expected output on non-symmetric images and kernels.
